$NetBSD: patch-aa,v 1.4 2011/12/09 12:42:10 dmcmahill Exp $

Add missing header file needed for setlocale() prototype as well as LC_ALL

--- src/main.c.orig	2011-09-17 21:29:29.000000000 +0000
+++ src/main.c
@@ -33,6 +33,10 @@
 #include "config.h"
 #endif
 
+#ifdef HAVE_LOCALE_H
+#include <locale.h>
+#endif
+
 #include <stdlib.h>
 #ifdef HAVE_STRING_H
 #include <string.h>
